### PR TITLE
Add Profile type

### PR DESCRIPTION
**Pull request by kumarsonsoff3**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [x] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/870"></a>

### CODE REVIEW

**Comment by Cahllagerfeld** [comment]
Thanks for your contribution 💥 
One tipp from my side: You shouldn't make your contributions on the main-branch of your fork. This might cause unwanted merge-conflicts in the future

The current action is failing due to a prettier-issue. Do you want to fix this, or do you want me to fix it for you?

**Comment by kumarsonsoff3** [comment]
Thanks! for the tipp, will remember!!
Could you fix this one for me!!

**Comment by kumarsonsoff3** [comment]
> Thanks for your contribution boom One tipp from my side: You shouldn't make your contributions on the main-branch of your fork. This might cause unwanted merge-conflicts in the future
> 
> The current action is failing due to a prettier-issue. Do you want to fix this, or do you want me to fix it for you?

@Cahllagerfeld , Thanks! for the tipp, will remember!!
Btw Could you fix this one for me!!

**Comment by kumarsonsoff3** [comment]
@Cahllagerfeld , Thanks! for the tipp, will remember it !!
Could you fix this one for me!!

**Comment by Cahllagerfeld** [comment]
Sure 👍

**Review by Cahllagerfeld — APPROVED** [review]
Looks good to me 👍

**Comment by kumarsonsoff3** [comment]
Thanks For your contribution man!! :+1:

**Comment by Cahllagerfeld** [comment]
You're welcome :D

File: public/data/kumarsonsoff3.json [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    },
+    {
+      "title": "Started Freelancing",
+      "date": "November 2021",
+      "icon": "dollar",
+      "color": "gold",
+      "description": "Started freelancing"
+    },
+    {
+      "title": "Published Firt blog on Hashnode",
```
Comment (Panquesito7) [inline public/data/kumarsonsoff3.json:34]: Minor spelling error here. Might want to create a good first issue for this. 🙂
```suggestion
      "title": "Published my first blog on Hashnode",
```

File: public/data/kumarsonsoff3.json [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+      "date": "October 2021",
+      "icon": "hashnode",
+      "color": "blue",
+      "description": "Published First blog on Hashnode"
```
Comment (Panquesito7) [inline public/data/kumarsonsoff3.json:38]:
```suggestion
      "description": "Published my first blog on Hashnode"
```

Comment (kumarsonsoff3) [reply public/data/kumarsonsoff3.json:38]: Thanks for your contribution to this fixing typo!!

**Pull request merged**